docs/presets/neutrino-preset-jest/README.md

```diff
@@ -144,6 +145,26 @@ Use the command line [`files` parameters](/cli/README.md#neutrino-test) to execu
 `neutrino-preset-jest` can watch for changes on your source directory and subsequently re-run tests. Simply use the
 `--watch` flag with your `neutrino test` command.
 
+## Coverage reporting
+
+Jest has an integrated coverage reporter, which requires no configuration. To collect test coverage information and
+generate report run:
+
+#### Yarn
+
+```bash
+❯ yarn test -- --coverage
```

Comment:
This assumes that they are running the command via a package script, which may not always be true. Let's change these particular scripts to:

```bash
❯ neutrino test --coverage
```

Then maybe mention something about a script target like:

```json
{
  "scripts": {
    "coverage": "neutrino test --coverage"
  }
}
```

Thoughts?